sql/parser/builtins.go

```diff
@@ -331,12 +334,23 @@ var builtins = map[string][]builtin{
 			types:      typeList{},
 			returnType: DummyFloat,
 			impure:     true,
-			fn: func(args DTuple) (Datum, error) {
+			fn: func(_ EvalContext, args DTuple) (Datum, error) {
 				return DFloat(rand.Float64()), nil
 			},
 		},
 	},
 
+	"unique_id": {
```

Comment:
K-sorted is obscure, but apparently the correct math term for "roughly sorted". `unique_id_roughly_sorted` or `roughly_sorted_unique_id` is even worse.